### DISCUSSION TITLE
How to play secured stream directly via RTMP link with available valid token ?

### DISCUSSION THREAD

**Discussion by ajayalag1974** [ant-media/Ant-Media-Server#5348] [Q&A]
Hi,

I can generate token & play secured stream via WebRTC but what if i directly wan't to play secured stream via some RTMP link ?

How i can do that ? 

I tried something like rtmp://antmedia.host/LiveApp/stream1?token=sometoken but its failing

**Comment by tremzo** [comment]
@ajayalag1974 you can append the token at the end of the stream key as detailed in this documentation for stream security https://antmedia.io/docs/guides/developer-sdk-and-api/rest-api-guide/stream-security/#how-to-use-jwt-token

```
<streamid>?token={tokenid}
```

**Reply by mekya** [reply]
hi @timantmedia ,
We don't maintain the RTMP playback and it's disabled by default. I think it also does not support secure streaming even if it's enabled.

**Reply by Mohit-3196** [reply]
Yes, RTMP playback is disabled by default. We recommend using [Dash play](https://antmedia.io/docs/guides/playing-live-stream/dash-playing-cmaf/) for similar latency of 3-5 seconds.
To enable RTMP playback, you can check this space https://github.com/orgs/ant-media/discussions/4874
Also, as mentioned it doesn't support secure streaming.

I hope it helps.

Thank you
